pom.xml

```diff
@@ -37,6 +37,7 @@
         <logbackVersion>1.0.13</logbackVersion>
         <resourceServerVersion>1.0.42</resourceServerVersion>
         <slf4jVersion>1.7.5</slf4jVersion>
+        <spring.version>3.2.13.RELEASE</spring.version>
```

Comment:
Love the refactoring out the `spring.version` rather than having the version repeated in 7 (7!) spots in the file. :smile: